src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,10 @@
 package core.basesyntax;
 
-public class ColorSupplier {
+import java.util.Random;
+
+public class ColorSupplier extends Ball {
```

Comment:
The ColorSupplier class should not extend the Ball class. The purpose of ColorSupplier is to provide a color, it is not a type of Ball. Please remove the inheritance from Ball.